metsim/io.py

```diff
@@ -8,21 +8,21 @@
 import struct
 import pandas as pd
 from configparser import ConfigParser
-import metsim
-from metsim.util import multi, method
+from metsim.forcing import Forcing
 
-
-def read_config(fpath, n_days=-1):
+def read_config(fpath: str, n_days=-1) -> ConfigParser:
     """
-    TODO
+    Reads the configuration file given.  If the
+    file does not exist, an empty ConfigParser
+    will be returned.
     """
     cfp = ConfigParser()
     if os.path.isfile(fpath):
         cfp.read(fpath)
     return cfp 
     
 
-def read_ascii(fpath, n_days=-1):
+def read_ascii(fpath, n_days=-1) -> Forcing:
```

Comment:
for this you can take a look here: https://github.com/jhamman/tonic/blob/master/tonic/models/vic/vic.py#L154